### DISCUSSION TITLE
Issue with  contribution chart. Wrong data being shown

### DISCUSSION THREAD

**Discussion by krishnaSpotnana** [community/community#87457] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The contribution chart is not being shown correctly. The old data seems to be missing. All the contributors commits decreased in last 2-3 weeks. I think the data for almost 1 and half  initial years is not being added. Earlier I was shown 700+ commits , now it shows 500+ commits.

The heatmap graph and the contribution chart do not match .

Heatmap graph: 
<img width="566" alt="Screenshot 2024-01-15 at 3 38 45 PM" src="https://github.com/community/community/assets/57883777/f7df71ed-37b4-4aea-80a0-f90b9eecdbb1">

Contribution chart: 

<img width="1028" alt="Screenshot 2024-01-15 at 3 39 01 PM" src="https://github.com/community/community/assets/57883777/51d881d6-fdb8-451a-8bc3-59ae3223762b">

The 2019 and 2020s data is missing in contribution chart.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @krishnaSpotnana ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by krishnaSpotnana** [reply]
Thanks for doing this @queenofcorgis ! Though, The issue is still not resolved . Tagging someone from the team would help?
